core/src/com/goonsquad/galactictd/systems/game/WarpGeneratingSystem.java

```diff
@@ -31,6 +31,6 @@ private void generateWarpLocation() {
         warpLocation.x = MathUtils.random(warpWidth, PlayScreen.GAME_WIDTH - warpWidth);
         warpLocation.y = MathUtils.random(
                 warpHeight,
-                PlayScreen.GAME_HEIGHT - PlayScreen.HUD_HEIGHT - warpHeight);
+                PlayScreen.GAME_HEIGHT - warpHeight);
```

Comment:
Oh okay, I see now. I must have misread because of the the line breaks then. Since they are roughly the same size, go ahead and format them the same (preferably on the same line, but your call).